core/station/impl/src/services/status.rs

```diff
@@ -0,0 +1,34 @@
+use crate::errors::StatusError;
+use ic_cdk::api::management_canister::main::{
+    self as mgmt, CanisterIdRecord, CanisterStatusResponse,
+};
+use lazy_static::lazy_static;
+use orbit_essentials::api::ServiceResult;
+use std::sync::Arc;
+
+lazy_static! {
+    pub static ref STATUS_SERVICE: Arc<StatusService> = Arc::new(StatusService::default());
+}
+
+#[derive(Default, Debug)]
+pub struct StatusService {}
```

Comment:
instead of creating granular services maybe we should just create a `ManagedCanisterService` and include there the `canister_status` call and `create_canister` that we introduced in a recent PR, wdyt?